mypy/semanal.py

```diff
@@ -3327,7 +3327,10 @@ def name_not_defined(self, name: str, ctx: Context) -> None:
     def name_already_defined(self, name: str, ctx: Context,
                              original_ctx: Optional[SymbolTableNode] = None) -> None:
         if original_ctx:
-            extra_msg = ' on line {}'.format(original_ctx.node.get_line())
+            if original_ctx.node and original_ctx.node.get_line() != -1:
```

Comment:
We're trying to catch the cases where the definition is repeated twice in the same module, and indicate the line where the first one occurred. I don't think `mod_id` would ever be a different module if it gets to this line, right? It could be either `None` (when the definition was added from an import perhaps) or the same as the current module.